src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java

```diff
@@ -464,6 +569,29 @@ public SELF containsSubsequence(double... subsequence) {
     return myself;
   }
 
+  /**
+   * Verifies that the actual array contains the given subsequence (possibly with other values between them).
+   * <p>
+   * Example:
+   * <pre><code class='java'> // assertion will pass
+   * assertThat(new double[] { 1.0, 2.0 }).containsSubsequence(new Double[] { 1.0, 2.0 });
+   * assertThat(new double[] { 1.0, 2.0, 2.0, 1.0 }).containsSubsequence(new Double[] { 1.0, 1.0 });
+   *
+   * // assertion will fail
+   * assertThat(new double[] { 1.0, 1.0, 2.0 }).containsSubsequence(new Double[] { 2.0, 1.0 });</code></pre>
+   *
+   * @param subsequence the subsequence of values to look for.
+   * @return myself assertion object.
+   * @throws AssertionError if the actual array is {@code null}.
+   * @throws AssertionError if the given array is {@code null}.
+   * @throws AssertionError if the actual array does not contain the given subsequence.
+   */
+  public SELF containsSubsequence(Double[] subsequence) {
```

Comment:
Please also add
```java
public SELF containsSubsequence(Double[] subsequence, Offset<Double> precision)
```

Reply:
https://github.com/assertj/assertj-core/blob/c0f48d3858e047ed0421a36c435e6f1b5503edcb/src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java#L430

It's already there

Reply:
https://github.com/assertj/assertj-core/blob/c0f48d3858e047ed0421a36c435e6f1b5503edcb/src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java#L490

It's already there